docs/guides/http-filters.md

```diff
@@ -131,3 +131,104 @@ type Response = {
   body?: string
 }
 ```
+
+## Modify Incoming Request Data
+
+You can also modify incoming request data by parsing and transforming the request's query parameters or body. Below is an example where we modify the incoming request data by extracting user information from a query parameter and then setting the request body with the transformed user data.
+
+```graphql
```

Comment:
Can we use a simpler example perhaps? May where the input us `{name: String, age: Int}` and the HTTP API needs it to be `{user_name: String, is_adult: Boolean}`.